File: src/main/java/io/github/chaosdave34/kitpvp/items/impl/tank/TankBoots.java
package io.github.chaosdave34.kitpvp.items.impl.tank;

import io.github.chaosdave34.kitpvp.abilities.Ability;
import io.github.chaosdave34.kitpvp.abilities.AbilityHandler;
import io.github.chaosdave34.kitpvp.items.CustomItem;
import net.kyori.adventure.text.Component;
import org.bukkit.Material;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public class TankBoots extends CustomItem {
    public TankBoots() {
        super(Material.DIAMOND_BOOTS, "tank_boots");
    }

    @Override
    public @NotNull Component getName() {
        return createSimpleItemName("Tank Boots");
    }

    @Override
    public @NotNull List<Ability> getAbilities() {
        return List.of(AbilityHandler.SEISMIC_WAVE);
    }
}

File: src/main/kotlin/io/github/chaosdave34/kitpvp/items/impl/tank/TankBoots.kt
package io.github.chaosdave34.kitpvp.items.impl.tank

import io.github.chaosdave34.kitpvp.abilities.Ability
import io.github.chaosdave34.kitpvp.abilities.AbilityHandler
import io.github.chaosdave34.kitpvp.items.CustomItem
import org.bukkit.Material

class TankBoots : CustomItem(Material.IRON_BOOTS, "tank_boots", "Tank Boots") {
    override fun getAbilities(): List<Ability> = listOf(AbilityHandler.GROUND_SLAM)
}